### PR TITLE
 Introduces basic Symbolizer Editor

### LINKED ISSUE #74: Implement a Rule GUI
Setup the UserInterface for a ``Rule``.

The final UI for the filter could look like this (somehow):

image

It consists of three key components:
- a ScaleDenominator
- a Filter
- a Symbolizer

### LINKED ISSUE #82: Implement a geometry style editor
In order to let the user create or edit the geometry style definition we need a ``StyleEditor`` component (naming not fixed). This component should offer tools to edit the style (fill-color, outline-color, etc.), to edit label placement and to show a preview map.

The UI could be inspired by:

image

### PR DESCRIPTION
**Pull request by KaiVolland**
This introduces a basic Symoblizer Editor. It adds support for editing `CircleSymbolizer`s.
It includes some Field Components which can be reused with other Symbolizer-Editors.

It also adds some minor changes to the RuleDemo.
symbolizer_editor

Closes #74 
Closes #82

### CODE REVIEW

File: src/Component/Symbolizer/CircleEditor/CircleEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          color={color}
+          label="Fill-Color"
+          onChange={(value: string) => {
+            symbolizer.color = value;
+            this.props.onSymbolizerChange(symbolizer);
+          }}
+        />
+        <OpacityField
+          opacity={opacity}
+          label="Fill-Opacity"
```
Comment (chrismayer) [inline src/Component/Symbolizer/CircleEditor/CircleEditor.tsx]: Labels should be put into pros, so they are configurable from ouside, e.g. for i18n.

File: src/Component/Symbolizer/CircleEditor/CircleEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          onChange={(value: number) => {
+            symbolizer.opacity = value;
+            this.props.onSymbolizerChange(symbolizer);
+          }}
+        />
+        <ColorField
+          color={strokeColor}
+          label="Stroke-Color"
```
Comment (chrismayer) [inline src/Component/Symbolizer/CircleEditor/CircleEditor.tsx]: See comment above.

File: src/Component/Symbolizer/CircleEditor/CircleEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+          onChange={(value: string) => {
+            symbolizer.strokeColor = value;
+            this.props.onSymbolizerChange(symbolizer);
+          }}
+        />
+        <WidthField
+          width={strokeWidth}
+          label="Stroke-Width"
```
Comment (chrismayer) [inline src/Component/Symbolizer/CircleEditor/CircleEditor.tsx]: Labels should be put into pros, so they are configurable from ouside, e.g. for i18n.

File: src/Component/Symbolizer/CircleEditor/CircleEditor.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+          onChange={(value: number) => {
+            symbolizer.strokeWidth = value;
+            this.props.onSymbolizerChange(symbolizer);
+          }}
+        />
+        <OpacityField
+          opacity={strokeOpacity}
+          label="Stroke-Opacity"
```
Comment (chrismayer) [inline src/Component/Symbolizer/CircleEditor/CircleEditor.tsx]: See comment above.

**Comment by marcjansen** [comment]
Cool, thanks everyone.

**Pull request merged**